### PR TITLE
Add storage

### PR DESCRIPTION
**Pull request by zi-hui**
create loadFromStorage() in main and addressing exit command at startup

### CODE REVIEW

File: src/main/java/seedu/kitchenhelper/KitchenHelper.java [hunk 1]
```diff
@@ -43,36 +43,34 @@ private void start() {
...
+        try {
+            ingredientList = new ArrayList<>(storage.getIngredientData());
+            recipeList = new ArrayList<>(storage.getRecipeData());
+            choreList = new ArrayList<>(storage.getChoreData());
+            storage.loadExpenditureData();
+        } catch (FileNotFoundException err) {
+            ingredientList = new ArrayList<>();
+            recipeList = new ArrayList<>();
+            choreList = new ArrayList<>();
+            Expenditure.getInstance().loadExpenditureVariables(0, 0, null);
```
Comment (jinfayap) [inline src/main/java/seedu/kitchenhelper/KitchenHelper.java:68]: Good job trying to extract them into a method.

File: src/main/java/seedu/kitchenhelper/ui/Ui.java [hunk 2]
```diff
@@ -71,11 +74,14 @@ public void validUserChoice(String userChoice) {
             System.out.println(MESSAGE_FOR_NORMAL_MODE);
         } else if (userChoice.trim().equals("2")) {
             System.out.println(MESSAGE_FOR_RESTORE_MODE);
+        } else if (userChoice.equalsIgnoreCase(ExitCommand.COMMAND_WORD)) {
```
Comment (jinfayap) [inline src/main/java/seedu/kitchenhelper/ui/Ui.java]: Would this be executed if `exit   ` was to be entered by the user?

Comment (zi-hui) [reply src/main/java/seedu/kitchenhelper/ui/Ui.java]: Nope it wont execute exit command, it shows message to ask users to enter a loading option first then re-enter exit

Comment (jinfayap) [reply src/main/java/seedu/kitchenhelper/ui/Ui.java]: @zi-hui  what i meant was, you never trim the user input. So when user type `exit[][][]` where [] is the spaces. will this statement be executed and showing the message involved in this if else statement?

Comment (zi-hui) [reply src/main/java/seedu/kitchenhelper/ui/Ui.java]: ok i made the change, thankss!!

**Pull request merged**